ui/src/composables/useRouteContext.ts

```diff
@@ -1,11 +1,14 @@
-import {Ref, watch} from "vue";
+import {ref, onMounted, watch, Ref} from "vue";
 import {useRoute} from "vue-router";
 
-export default function useRouteContext(routeInfoTitle: Ref<string>, embed: boolean = false) {
+export default (routeInfo: Ref<{title:string}>) => {
```

Comment:
NOTE: we never use anyother members of this `routeInfoObject` so why not use a string instead?